scss/functions/_get-color.scss

```diff
@@ -1,10 +1,18 @@
 @use '../variables' as *;
 @use 'sass:map';
 
-@function get-color($type, $level) {
-  // add the logic here
-  // use if else to check if the type and level are valid
-  // if they are valid, return the color
-  // if they are not valid warn 'Invalid color type or level' and return null
-  // null is a special value in Sass that means nothing, and it is used to represent the absence of a value, you need to return null if the type or level is not valid
+// add the logic here
+// use if else to check if the type and level are valid
+// if they are valid, return the color
+// if they are not valid warn 'Invalid color type or level' and return null
+// null is a special value in Sass that means nothing, and it is used to represent the absence of a value, you need to return null if the type or level is not valid
+
+@function color($type, $level) {
+  @if map.has-key($types, $type) and map.has-key($levels, $level) {
+    @warn 'type or level valid';
+  } @else {
+    @warn 'Invalid color type or level';
+    // @return null;
+  }
 }
+@include color('brand', '900');
```

Comment:
`@function get-color($type, $level) {
  // add the logic here
  // use if else to check if the type and level are valid
  // if they are valid, return the color
  // if they are not valid warn 'Invalid color type or level' and return null
  // null is a special value in Sass that means nothing, and it is used to represent the absence of a value, you need to return null if the type or level is not valid

}`
untuk nama `@function` nya ikuti seperti contoh saya di atas ya, sesuai soal pada awalnya. 

untuk line 18 baiknya dihapus saja ya